src/test/java/com/nvidia/spark/rapids/jni/CastStringsTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.AssertUtils;
+import ai.rapids.cudf.ColumnVector;
+import ai.rapids.cudf.DType;
+import ai.rapids.cudf.Table;
+import org.junit.jupiter.api.Test;
+
+import java.math.RoundingMode;
+import java.util.stream.IntStream;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CastStringsTest {
+  @Test
+  void castToIntegerTest() {
+    Table.TestBuilder tb = new Table.TestBuilder();
+    tb.column(3l, 9l, 4l, 2l, 20l, null);
+    tb.column(5, 1, 0, 2, 7, null);
+    tb.column(new Byte[]{2, 3, 4, 5, 9, null});
```

Comment:
Nit: It'd be nice to add an input that expects the cast to generate a null that wasn't in the input to make sure validity of the result is plumbed through properly.

Reply:
an invalid string, you mean?

Reply:
yes, an invalid string to cast to an integer, should generate null.